### PR TITLE
Form verification

### PR DESCRIPTION
**Pull request by OmarG247**

### CODE REVIEW

File: src/pages/registration.tsx [hunk 1]
```diff
@@ -602,12 +618,16 @@ function Registration() {
...
-        }
-      });
+      if (formIsVerified()) {
+        registerUser(userForm, auth.accessToken?.token ?? '').then((res) => {
+          if (res === 'Created') {
+            setRegistered(true);
+          }
+        });
+      } else {
+        alert('Please complete all mandatory fields');
```
Comment (dellisd) [inline src/pages/registration.tsx:628]:
```suggestion
        alert('Please fill out all required fields');
```

**Review by dellisd — CHANGES_REQUESTED** [review]

File: src/pages/registration.tsx [hunk 2]
```diff
@@ -236,20 +236,36 @@ function Registration() {
   const finalStep = steps.additional;
   const history = useHistory();
 
+  useEffect(() => {
+    console.log(userForm);
+  }, [userForm]);
+
```
Comment (Walsker) [inline src/pages/registration.tsx:242]: Remove debug info
```suggestion
```

**Review by Walsker — CHANGES_REQUESTED** [review]
Hover effect still persists when disabled.

image
image

File: src/pages/registration.tsx (hunk 2 above)
Comment (OmarG247) [reply src/pages/registration.tsx:242]: Updating

File: src/pages/registration.tsx (hunk 1 above)
Comment (OmarG247) [reply src/pages/registration.tsx:628]: Updating

**Review by dellisd — APPROVED** [review]

**Review by Walsker — APPROVED** [review]
The cursor effect is still there, but I'll remove it later

**Pull request merged**